### PR TITLE
Do not allow underscores in docker cluster names. Also install C* python driver without extensions when creating the docker image

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: tool/cstar_perf/docker/cstar_docker.py [hunk 1]
```diff
@@ -145,6 +145,9 @@
     echo "autorestart=true"                                                                 >> /supervisord.conf && \
     echo "redirect_stderr=true"                                                             >> /supervisord.conf
 
+### install the C* driver without any extensions to speed up installation time
+RUN CASS_DRIVER_NO_EXTENSIONS=1 pip install cassandra-driver
```
Comment (nastra) [inline tool/cstar_perf/docker/cstar_docker.py]: that way we don't need to install the driver when creating docker containers for the cluster + the frontend. Also installing it without extensions speeds up the installation process drastically

**Comment by nastra** [comment]
Don't merge yet. Looks like this causes the bootstrapping to fail

````
172.17.0.4] out: [cstar@test-frontend-00] out: jar:
[172.17.0.4] out: [cstar@test-frontend-00] out:      [copy] Copying 1 file to /home/cstar/fab/cassandra/build/classes/main/META-INF
[172.17.0.4] out: [cstar@test-frontend-00] out:      [copy] Copying 1 file to /home/cstar/fab/cassandra/build/classes/thrift/META-INF
[172.17.0.4] out: [cstar@test-frontend-00] out:      [copy] Copying 1 file to /home/cstar/fab/cassandra/build/classes/main/META-INF
[172.17.0.4] out: [cstar@test-frontend-00] out:      [copy] Copying 1 file to /home/cstar/fab/cassandra/build/classes/thrift/META-INF
[172.17.0.4] out: [cstar@test-frontend-00] out:       [jar] Building jar: /home/cstar/fab/cassandra/build/apache-cassandra-thrift-2.2.6-SNAPSHOT.jar
[172.17.0.4] out: [cstar@test-frontend-00] out:       [jar] Building jar: /home/cstar/fab/cassandra/build/apache-cassandra-2.2.6-SNAPSHOT.jar
[172.17.0.4] out: [cstar@test-frontend-00] out:       [jar] Building jar: /home/cstar/fab/cassandra/build/apache-cassandra-clientutil-2.2.6-SNAPSHOT.jar
[172.17.0.4] out: [cstar@test-frontend-00] out:     [mkdir] Created dir: /home/cstar/fab/cassandra/build/classes/stress/META-INF
[172.17.0.4] out: [cstar@test-frontend-00] out:     [mkdir] Created dir: /home/cstar/fab/cassandra/build/tools/lib
[172.17.0.4] out: [cstar@test-frontend-00] out:       [jar] Building jar: /home/cstar/fab/cassandra/build/tools/lib/stress.jar
[172.17.0.4] out: [cstar@test-frontend-00] out:
[172.17.0.4] out: [cstar@test-frontend-00] out: BUILD SUCCESSFUL
[172.17.0.4] out: [cstar@test-frontend-00] out: Total time: 1 minute 7 seconds
[172.17.0.4] out: [cstar@test-frontend-00] out:
[172.17.0.4] out:
[172.17.0.4] out: [cstar@test-frontend-00] run: cp -a ~/fab/cassandra ~/fab/cassandra_builds/a7975f385681e0ae17bc0fea0260065142b85fd9
[172.17.0.4] out: [cstar@test-frontend-00] run: ls -1 ~/fab/cassandra_builds | wc -l
[172.17.0.4] out: [cstar@test-frontend-00] out: 1
[172.17.0.4] out: [cstar@test-frontend-00] out:
[172.17.0.4] out:
[172.17.0.4] out:
[172.17.0.4] out: Fatal error: One or more hosts failed while executing task 'bootstrap'
[172.17.0.4] out:
[172.17.0.4] out: Aborting.
[172.17.0.4] out:


Fatal error: run() received nonzero return code 1 while executing!

Requested: cstar_perf_bootstrap -v cassandra-2.2.6
Executed: /bin/bash -l -c "cstar_perf_bootstrap -v cassandra-2.2.6"

Aborting.
INFO:cstar_docker:Destroying test_frontend containers...
```

**Comment by nastra** [comment]
Pushed a newer version where we don't allow **underscores** in the docker cluster names. I will update the cstar_perf + docker documentation in a separate PR.

**Comment by mshuler** [comment]
lgtm

**Pull request merged**